Sources/Widgets/Widgets3D/ResliceCursorWidget/cprBehavior.js

```diff
@@ -0,0 +1,116 @@
+import resliceCursorBehavior from 'vtk.js/Sources/Widgets/Widgets3D/ResliceCursorWidget/behavior';
+import {
+  InteractionMethodsName,
+  planeNameToViewType,
+} from 'vtk.js/Sources/Widgets/Widgets3D/ResliceCursorWidget/Constants';
+import { updateState } from 'vtk.js/Sources/Widgets/Widgets3D/ResliceCursorWidget/helpers';
+import { ViewTypes } from 'vtk.js/Sources/Widgets/Core/WidgetManager/Constants';
+import { vec3 } from 'gl-matrix';
+// import macro from 'vtk.js/Sources/macros';
+
+export default function widgetBehavior(publicAPI, model) {
+  // We inherit resliceCursorBehavior
+  resliceCursorBehavior(publicAPI, model);
+  const superClass = { ...publicAPI };
+
+  model.stretchPlaneName = 'Y';
+  model.stretchPlaneViewType = planeNameToViewType[model.stretchPlaneName];
+  model.crossPlaneName = 'Z';
+  model.crossPlaneViewType = planeNameToViewType[model.crossPlaneName];
+  model.previousData = null;
+
+  publicAPI.getActiveInteraction = () => {
+    if (
+      model.widgetState
+        .getStatesWithLabel(`lineIn${model.stretchPlaneName}`)
+        .includes(model.activeState)
+    ) {
+      return InteractionMethodsName.TranslateCenterAndUpdatePlanes;
```

Comment:
I though you could return `TranslateLine` and make RCW/behavior.js support a manipulator that returns a `worldDirection`. 
Wouldn't it work ?

Reply:
I could eventually use InteractionMethodsName.TranslateCenter but it actually doing:
`newCenter = currentCenter + currentWorldCoords - previousWorldCoords`
Which is not suitable for the CPR

Reply:
Can't `newCenter = currentCenter + currentWorldCoords - previousWorldCoords` be moved in the plane manipulator ?

Reply:
This would mean that the PlaneManipulator doesn't return the clicked position anymore, which is a breaking change and many widgets rely on this manipulator
Furthermore, a manipulator shouldn't be stateful (the cprManipulator being an exception now)

Reply:
Ok to leave it as is for now. It can be simplified in a different PR.